i18n/en.json

```diff
@@ -2050,15 +2050,15 @@
     "translation": "LTI-19: Failed to login user."
   },
   {
-    "id": "edx.build_user.email_missing",
+    "id": "lms.build_user.email_missing",
     "translation": "LTI-23: LTI Launch Data is missing Email."
   },
   {
-    "id": "edx.build_user.username_missing",
+    "id": "lms.build_user.username_missing",
     "translation": "LTI-24: LTI Launch Data is missing Username."
   },
   {
-    "id": "edx.build_user.lti_user_id_missing",
+    "id": "lms.build_user.lti_user_id_missing",
```

Comment:
Just checking if you verified this change. I made the name change, but I hadn't really settled on how or what it should be, and so I hadn't checked that I changed all uses to match. I just remember constantly changing the names in the sources as different names seemed better.